src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return 40 * days;
+  }
+
+  if (days >= 3 && days < 7) {
+    return 40 * days - 20;
+  }
+
+  if (days >= 7) {
+    return 40 * days - 50;
+  }
```

Comment:
use else if, if you have more then one if.

Reply:
Is it really needed if u immediately return the value? As for me - no